commands/viewer_commands.py

```diff
@@ -141,26 +141,37 @@ async def bet(
             Choice(name="Down", value="Down"),
             Choice(name="R", value="R"),
             Choice(name="L", value="L"),
+        ],
+        amount=[
+            Choice(name="1", value=1),
+            Choice(name="2", value=2),
+            Choice(name="3", value=3),
+            Choice(name="4", value=4),
+            Choice(name="5", value=5),
+            Choice(name="6", value=6),
+            Choice(name="7", value=7),
+            Choice(name="8", value=8),
+            Choice(name="9", value=9)
         ]
     )
-    async def pokemon_move(self, interaction: Interaction, move: str):
-        """Send a move to the Pokemon game"""
+    async def pokemon_move(self, interaction: Interaction, move: str, amount: int):
+        """Send Pokemon move data to frontend"""
```

Comment:
The description is what the USER sees in Discord - they aren't concerned or aware of the entire data and frontend stuff - I would leave the prior comment.